snuba/schemas.py

```diff
@@ -1,9 +1,16 @@
 import copy
 from datetime import datetime, timedelta
-from typing import Any, Mapping
+from typing import Any, Mapping, MutableMapping
 
 import jsonschema
 
+from snuba import environment
+from snuba.utils.metrics.backends.wrapper import MetricsWrapper
+from snuba.utils.metrics.decorators import track_calls
+
+
+timeseries_metrics = MetricsWrapper(environment.metrics, "extensions.timeseries")
+
 
 def get_time_series_extension_properties(
```

Comment:
Also, this function seems to be in the wrong file: it should be in `snuba.query.timeseries`, assuming `snuba.query.{project,organization}_extension` have all of their contents in the right place (and `snuba.query.timeseries` itself should probably be `snuba.query.timeseries_extension` at that.)

Reply:
Ah, that was was probably something that I missed during that cleanup then. I'll move this in a follow up.